neo/public/javascripts/common/services.js

```diff
@@ -1,12 +1,157 @@
 angular.module('cloudberry.common', [])
   .service('Asterix', function($http, $timeout, $location) {
```

Comment:
we've blaming for this `Asterix` name for a long time. could you fix it by this time by correctly name this service `Cloudberry`? thanks!